### ISSUE TITLE
引用svg文件时，会报Expression expected

### ISSUE THREAD

**Issue by linzhaoyang** [alibaba/ice#6628]
### Describe the bug

<img width="475" alt="企业微信截图_16992519571527" src="https://github.com/alibaba/ice/assets/38873920/8bc4c933-cd04-4e7b-b548-0b13c37d8ef8"> 使用import加载svg文件会直接报错，在ice2.x时，可以直接使用。

### Expected behavior

svg文件能正确通过import引用并通过img的src属性加载

### Version of ice.js

^3.3.0

### Content of build.json or ice.config.mts

```TypeScript
import { defineConfig } from '@ice/app';
import antd from '@ice/plugin-antd';
import store from '@ice/plugin-store';
import request from '@ice/plugin-request';
import routes from './routes';
import appProvider from './plugins/appProvider/index';
import { modifyLoader } from '@ice/webpack-modify';
// The project config, see https://v3.ice.work/docs/guide/basic/config

export default defineConfig(() => ({
  dataLoader: false,
  syntaxFeatures: { exportDefaultFrom: true },
  compileDependencies: true,
  ssg: false, // 关闭ssg, 不然procomontents、react-activation等不支持ssg的框架、插件会报错
  hash: true,
  plugins: [
    antd({
      importStyle: true,
    }),
    store({ resetPageState: true }),
    request(),
    appProvider(),
  ],
   routes: {
    ignoreFiles: [
     '**',
    ],
    config: routes,
  },
}));
```

**Comment by ClarkXia** [comment]
package.json 信息也需要提供下

**Comment by linzhaoyang** [comment]
> package.json 信息也需要提供下

{
  "name": "@ice/lite-scaffold",
  "version": "1.2.0",
  "description": "A new ice.js project.",
  "dependencies": {
    "@ant-design/colors": "^7.0.0",
    "@ant-design/compatible": "^5.1.2",
    "@ant-design/cssinjs": "^1.17.2",
    "@ant-design/icons": "^5.2.6",
    "@ant-design/pro-components": "^2.6.35",
    "@antv/g2": "^5.1.6",
    "@antv/layout": "^0.3.24",
    "@antv/x6": "^2.15.3",
    "@antv/x6-plugin-keyboard": "^2.2.1",
    "@antv/x6-plugin-minimap": "^2.0.6",
    "@antv/x6-plugin-scroller": "^2.0.10",
    "@antv/x6-plugin-selection": "^2.2.1",
    "@antv/x6-react-shape": "^2.2.2",
    "@dnd-kit/core": "^6.0.8",
    "@dnd-kit/sortable": "^7.0.2",
    "@dnd-kit/utilities": "^3.2.1",
    "@ice/runtime": "^1.2.7",
    "@linzhaoyang/react-activation": "^1.0.3",
    "antd": "^5.11.0",
    "dagre": "^0.8.5",
    "dayjs": "^1.11.10",
    "fabric": "^5.3.0",
    "html2canvas": "^1.4.1",
    "pdfjs-dist": "^4.0.189",
    "rc-virtual-list": "^3.11.3",

**Comment by linzhaoyang** [comment]
> package.json 信息也需要提供下

{
  "name": "@ice/lite-scaffold",
  "version": "1.2.0",
  "description": "A new ice.js project.",
  "dependencies": {
    "@ant-design/colors": "^7.0.0",
    "@ant-design/compatible": "^5.1.2",
    "@ant-design/cssinjs": "^1.17.2",
    "@ant-design/icons": "^5.2.6",
    "@ant-design/pro-components": "^2.6.35",
    "@antv/g2": "^5.1.6",
    "@antv/layout": "^0.3.24",
    "@antv/x6": "^2.15.3",
    "@antv/x6-plugin-keyboard": "^2.2.1",
    "@antv/x6-plugin-minimap": "^2.0.6",
    "@antv/x6-plugin-scroller": "^2.0.10",
    "@antv/x6-plugin-selection": "^2.2.1",
    "@antv/x6-react-shape": "^2.2.2",
    "@dnd-kit/core": "^6.0.8",
    "@dnd-kit/sortable": "^7.0.2",
    "@dnd-kit/utilities": "^3.2.1",
    "@ice/runtime": "^1.2.7",
    "@linzhaoyang/react-activation": "^1.0.3",
    "antd": "^5.11.0",
    "dagre": "^0.8.5",
    "dayjs": "^1.11.10",
    "fabric": "^5.3.0",
    "html2canvas": "^1.4.1",
    "pdfjs-dist": "^4.0.189",
    "rc-virtual-list": "^3.11.3",
    "react": "^18.2.0",
    "react-countup": "^6.5.0",
    "react-dom": "^18.2.0",
    "react-resizable": "^3.0.5",
    "react-sticky-box": "^2.0.5",
    "store": "^2.0.12"
  },
  "devDependencies": {
    "@applint/spec": "^1.2.3",
    "@ice/app": "^3.3.0",
    "@ice/plugin-auth": "^1.0.2",
    "@ice/plugin-request": "^1.0.2",
    "@ice/plugin-store": "^1.1.2",
    "@ice/webpack-modify": "^1.0.1",
    "@types/node": "^18.18.8",
    "@types/react": "^18.0.0",
    "@types/react-dom": "^18.0.0",
    "eslint": "^8.35.0",
    "stylelint": "^15.2.0",
    "svg-sprite-loader": "^6.0.11",
    "typescript": "^4.9.5"
  },"scripts": {
    "start": "ice start --mode development  --speedup",
    "start:uat": "ice start --mode uat",
    "start:prod": "ice start --mode prod",
    "build:dev": "ice build --mode development",
    "build:uat": "ice build --mode uat",
    "build:prod": "ice build --mode prod",
    "eslint": "eslint ./src --cache --ext .js,.jsx,.ts,.tsx",
    "eslint:fix": "npm run eslint -- --fix",
    "stylelint": "stylelint \"src/**/*.{css,scss,less}\" --cache",
    "stylelint:fix": "npm run stylelint -- --fix"
  },
  "publishConfig": {
    "access": "public"
  },
  "repository": "git@github.com:ice-lab/react-materials.git"
}

**Comment by ClarkXia** [comment]
@linzhaoyang 本周会修复，暂时可以通过移除 start 中的 --speedup 解决

**Comment by ClarkXia** [comment]
fixed in 3.3.8
